src/components/CampaignOverview.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <section class="favor-ranking">
+    <table class="favor-ranking-table tableFixHead">
```

Comment:
I think at some point we will need to make the table responcive so it would look good on all kind of screens.
Maybe it would be easier to use grid or flexboxes but I'm not sure.
But for now I think it's fine.